plugins/Kaleidoscope-LEDEffect-BootAnimation/src/kaleidoscope/plugin/LEDEffect-BootAnimation.cpp

```diff
@@ -15,7 +15,7 @@
  * this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "Kaleidoscope/plugin/LEDEffect-BootAnimation.h"
+#include "Kaleidoscope-LEDEffect-BootAnimation.h"
```

Comment:
nit: I would fix the capitalization error, instead of using the `Kaleidoscope-` version, for consistency with other plugins.

Reply:
Oops. I'd thought I *was* going with house style. thanks.